src/Commands/RunScan.ts

```diff
@@ -29,6 +29,12 @@ export class RunScan implements CommandModule {
         requiresArg: true,
         demandOption: true
       })
+      .option('auth', {
+        alias: 'o',
```

Comment:
Hm... Could you explain your choice of alias?

Reply:
`-o` for auto **object**
`a` id already used